### ISSUE TITLE
MavenLauncher NPE using pom.xml with no 'build' section

### ISSUE THREAD

**Issue by fvarrebola** [INRIA/spoon#1716]
According to the [Maven XSD](http://maven.apache.org/xsd/maven-4.0.0.xsd) the 'build' section of a 'pom.xml' file is optional. 

```
<xs:element minOccurs="0" name="build" type="Build">
```
However, if such a file is passed as input to the MavenLauncher, the following NPE exception is thrown:

```
Exception in thread "main" java.lang.NullPointerException
    at spoon.MavenLauncher$InheritanceModel.getSourceVersion(MavenLauncher.java:344)
    at spoon.MavenLauncher.<init>(MavenLauncher.java:113)
    at spoon.MavenLauncher.<init>(MavenLauncher.java:59)
```

**Comment by surli** [comment]
Hi @fvarrebola thanks for reporting this! The fix should be available in next Spoon 6.0.0 release.

**Comment by monperrus** [comment]
closed per #1716
